src/components/QuestionPlay/index.tsx

```diff
@@ -8,6 +8,7 @@ import WithOptions from './WithOptions';
 import { useStore } from '../../useStore';
 import styles from './styles.module.css';
 import classNames from 'classnames';
+import { getBadgeColor } from '../../helpers/common';
```

Comment:
We should not put it in common helper. Maybe questionPointsHelper or something?

Reply:
added quizViewHelper